sdks/api-clients/apivideo-nodejs-client.md

```diff
@@ -247,6 +248,22 @@ Method | Description | HTTP request
  - [WebhooksListResponse](https://github.com/apivideo/api.video-nodejs-client/blob/main/docs/model/WebhooksListResponse.md)
 
 
+### Rate Limiting
+
+api.video implements [rate limiting](https://docs.api.video/reference#limitation) to ensure fair usage and stability of the service. The rate limit values can be retrieved from the response headers of any API request. 
```

Comment:
We should probably clarify that it does not apply to `/auth` routes. So maybe something like this?

> The API provides the rate limit values in the response headers for any API requests you make. The [`/auth`](https://docs.api.video/reference/api/Advanced-authentication) endpoint is the only route without rate limitation.